airflow/contrib/kubernetes/kube_client.py

```diff
@@ -48,4 +48,10 @@ def _load_kube_config(in_cluster, cluster_context, config_file):
 def get_kube_client(in_cluster=conf.getboolean('kubernetes', 'in_cluster'),
                     cluster_context=None,
                     config_file=None):
+    if cluster_context is None:
+        if conf.get('kubernetes', 'cluster_context'):
```

Comment:
Merging this if statement with its parent on line 51 would reduce nesting and (in my opinion) improve readability.

Reply:
@pgagnon Thanks. Incorporated comments